vars/buildWDKAutoSwitch.groovy

```diff
@@ -150,6 +150,7 @@ def call(Map config = [unityVersions:[]]) {
                     nunit failIfNoResults: false, testResultsPattern: '**/build/reports/unity/**/*.xml'
                     archiveArtifacts artifacts: '**/build/logs/**/*.log', allowEmptyArchive: true
                     archiveArtifacts artifacts: '**/build/reports/unity/**/*.xml' , allowEmptyArchive: true
+                    publishCoverage adapters: [istanbulCoberturaAdapter('**/codeCoverage/Cobertura.xml')], sourceFileResolver: sourceFiles('NEVER_STORE')
```

Comment:
Why do we want the never store option?

Reply:
I was not sure if I already wanted to store source code. I guess for the WDKs it's fine, since they are quite small and it makes sense to store at the least the last build.

Reply:
@Larusso ok switched it to keep last build. That makes more sense.

Reply:
good.